gems/smithy/lib/smithy/anvil/client/views/module.rb

```diff
@@ -38,7 +38,8 @@ def requires
             if @plan.type == :types
               [:types]
             else
-              %i[client errors types]
+              # Order matters here - plugins must come before client
```

Comment:
This is fine. Perhaps we should break this into two methods, plugins and components. This list will get long.

Reply:
yeah that makes sense.  I'll leave to a future refactoring outside of this PR.